docs/modular-resources/configure.md

```diff
@@ -12,19 +12,21 @@ tags:
     "components",
     "services",
   ]
-description: "Add and configure a module from the Viam registry on your robot."
+description: "Add a modular resource to your robot by configuring it."
 no_list: true
 aliases:
   - "/program/extend/modular-resources/configure/"
 ---
 
-You can extend Viam by adding a module on your robot that provides one or more {{< glossary_tooltip term_id="resource" text="modular resources" >}} ([components](/components/) or [services](/services/)).
+To use a [modular resource](/modular-resources/) on your robot, you need to:
 
-You can [add a module from the Viam registry](#add-a-module-from-the-viam-registry), or you can [add a local module](#local-modules).
+1. Add a {{< glossary_tooltip term_id="module" text="module" >}}, either one [from the registry](#add-a-modular-resource-from-the-viam-registry) or a [local module](#local-modules).
+   This makes the modular resource available to the robot.
+1. Then add the modular resource itself.
 
-See [Key Concepts of Modular Resource APIs](/modular-resources/key-concepts/) for more information.
+These two steps automatically happen at the same time when you add a modular resource from the registry.
```

Comment:
What do you think of this rephrase:

```suggestion
When you add a modular resource from the registry, the underlying module that provides it is automatically added at the same time.
To add a modular resource from a local module, you must add the module first.
```

- "is automatically added" rather than "time" so we explicitly note the (absence of) user agency: no action required for the underlying module when from registry.
- Bit more explicit on what to do for each path to adding a modular resource, especially since we just introduced the two just above. Now we state how to initially interact with modules for both as well.

Reply:
good points. I like it.